Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.1.0"
 edition = "2021"
 
 [[bin]]
-name = "sara"
+name = "opendid"
 path = "src/main.rs"
```

Comment:
these three lines can get removed. That should be the default name since the package is called `opendid`.

Reply:
7-9?